README.md

```diff
@@ -75,7 +28,62 @@ If set up correctly, and assuming you have internet access and the Youtube API i
 * [webpacker](https://github.com/rails/webpacker)
 * [selenium-webdriver](https://www.seleniumhq.org/docs/03_webdriver.jsp)
 * [chromedriver-helper](http://chromedriver.chromium.org/)
+* [PostgreSQL](https://www.postgresql.org/)
+* [TravisCI](https://travis-ci.org/)
+
+### Gems
+- rspec-rails
+- capybara
+- launchy
+- pry
+- simplecov
+- shoulda-matchers
+- factory_bot_rails
+- orderly
+- vcr
+- webmock
+- faker
+- rubocop
+- figaro
+
+# Getting Started
+
+## Installing
+
+#### Clone repository:
+```javascript
+git clone git@github.com:janegreene/brownfield_of_dreams.git
+```
+#### Install gems:
+```javascript
+bundle install
+```
+#### Configure databases:
+```javascript
+rails db:{create,migrate,seed}
+```
+#### Fire up local server: (http://localhost:3000)
+```javascript
+rails s
+```
+#### Run test suite:
+```javascript
+bundle exec rspec
+```
 
-### Versions
```

Comment:
Could still have these. Have code from my last project that made little pictures for the Ruby and Rails logos, even